lib/utils/isStandardSyntaxValue.js

```diff
@@ -6,18 +6,26 @@ const hasInterpolation = require("../utils/hasInterpolation")
  * Check whether a value is standard
  */
 module.exports = function (value/*: string*/)/*: boolean*/ {
+  let normalizedValue = value
+
+  // Ignore signs before variable (example `-$variable`)
```

Comment:
Maybe `Ignore operators before variables (example `-$variable`)` instead of `signs`?